react-components/src/components/RuleBasedOutputs/RuleBasedOutputs.tsx

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { useEffect, type ReactElement } from 'react';
+
+import { type CogniteCadModel } from '@cognite/reveal';
+import { useAllMappedEquipmentAssetMappings, useReveal } from '../..';
+import { Color } from 'three';
+import { type RuleOutputSet } from './types';
+import { generateRuleBasedOutputs } from './utils';
+
+export type ColorOverlayProps = {
+  ruleSet: RuleOutputSet | undefined;
+};
+
+export function RuleBasedOutputs({ ruleSet }: ColorOverlayProps): ReactElement | undefined {
```

Comment:
Can we rename this to e.g. `RuleBasedOutputsSelector`?

Reply:
yep. Done
https://github.com/cognitedata/reveal/pull/4210/commits/e4c1637a2c6f4412f8fa79861d3d37844a646860